### ISSUE TITLE
Multiple file uploads break on editing job listing

### ISSUE THREAD

**Issue by stevenkellow** [Automattic/WP-Job-Manager#857]
When using the custom job application field for uploading files, if the job listing is edited after being published - the file uploads section will return a string of "Array".

Here is what a var dump of `get_post_meta( $post->ID, '_job_files', true );` returns under different circumstances:

### Single Document

array(1) { [0]=> string(113) "http://www.example.com/wp-content/uploads/job-manager-uploads/job_files/2017/01/document1.docx" }

### Multi Documents - listing not edited

array(3) { [0]=> string(119) "http://www.example.com/wp-content/uploads/job-manager-uploads/job_files/2017/01/document1.pdf" [1]=> string(116) "http://www.example.com/wp-content/uploads/job-manager-uploads/job_files/2016/09/document2.pdf" [2]=> string(111) "http://www.example.com/wp-content/uploads/job-manager-uploads/job_files/2017/01/document3.docx" }

### Multiple Documents - listing edited

string(5) "Array"

This error occurs on the latest version of WP Job Manager (1.25.2) and has occurred in the past.  The error appears whether using Ajax to upload the files or not.

**Comment by spencerfinnell** [comment]
Are you using WordPress 4.7.1? 

Can you upload those files normally via "Media ▸ Add New"?

See: https://core.trac.wordpress.org/ticket/39550

**Comment by stevenkellow** [comment]
Yes, WordPress 4.7.1 and adding Media is working fine.

**Comment by tripflex** [comment]
@stevenkellow what addon plugins and theme are you using?

**Comment by tripflex** [comment]
Also you said "When using the custom job application field for uploading files"

Are you referring to only some type of job application field, or any file upload field?  This comment confuses me in regards to what your issue is

**Comment by stevenkellow** [comment]
I'm using WP Job Manager - Application Deadline, WP Job Manager - Contact Listing, WP Job Manager add-on - Jobs Geolocation plugins as well, and I'm using a child theme of [Corporate Lite](https://en-gb.wordpress.org/themes/corporate-lite/).

I'm using a file upload field to modify the application form, so the code for it is:

```
$fields['job']['job_files'] = array(
	'label'       => __( 'Job/Person Specification', 'text-domain' ),  
	'description' => 'Files outlining further information about the position',  
	'type'        => 'file',  
	'required'    => false,  
	'placeholder' => '',  
	'priority'    => 5.3,  
	'ajax'        => false,  
	'multiple'    => true,  
	'allowed_mime_types' => array(  
		'pdf'  => 'application/pdf',  
		'doc'  => 'application/msword',  
		'docx' => 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',  
		'odt'  => 'application/vnd.oasis.opendocument.text',  
                'rtf'  => 'application/rtf',  
	)  
);
```

**Comment by tripflex** [comment]
I'm unable to reproduce this issue with standard twenty twelve install, using your same exact config through the filter, with two files, a doc and docx file

Have you tested this on a clean install yourself, or by switching to different theme?  I think this may be related to your custom code, theme or other plugins

**Comment by jom** [comment]
Should be fixed in WordPress 4.7.3. 

https://core.trac.wordpress.org/ticket/39550
